Services/CO.CDP.DataSharing.WebApi.Tests/AutoMapper/AutoMapperFixture.cs

```diff
@@ -0,0 +1,13 @@
+using AutoMapper;
+using CO.CDP.DataSharing.WebApi.AutoMapper;
+
+namespace CO.CDP.DataSharing.Tests.AutoMapper;
+
+public class AutoMapperFixture
```

Comment:
This fixture is meant to be used with a test case. The test case is missing. Have a look at other projects. The test case ensures that automapper config can be loaded.

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/a31c9205266b21418aa39449f1ecbc49cc58afab/Services/CO.CDP.Tenant.WebApi.Tests/AutoMapper/WebApiToPersistenceProfileTest.cs#L3-L10

Reply:
done.